app/gateway/sbom.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "{{site.base_gateway}} software bill of materials"
+content_type: policy
+layout: reference
+
+products:
+  - gateway
+
+breadcrumbs:
+  - /gateway/
+
+description: |
+  Kong provides a software bill of materials (SBOM) for every minor release, starting with 3.3.0.0.
+
+related_resources:
+  - text: "Secure {{site.base_gateway}}"
+    url: /gateway/security/
+  - text: Kong vulnerability patching process
+    url: /gateway/vulnerabilities/
+  - text: "Supported third-party dependencies for {{site.base_gateway}}"
+    url: /gateway/third-party-support/
+---
+
+A software bill of materials (SBOM) is an inventory of all software components (proprietary and open-source), open-source licenses, and dependencies in a given product. A software bill of materials (SBOM) provides visibility into the software supply chain and any license compliance, security, and quality risks that may exist.
+
+Starting in {{site.ee_product_name}} 3.3, we generate SBOMs for our artifact images.
+
+See the following links to directly download the SBOM for your {{site.ee_product_name}} version:
```

Comment:
```suggestion
```
This might be a bad suggestion so feel free to ignore it. I think that this sentece doesn't need to exist and we can call the title of the second column "Direct Download link"